llmlite/llms/chatglm2.py

```diff
@@ -16,19 +17,22 @@ class ChatGLMChat(Chat):
     def __init__(
         self,
         model_name_or_path: str,
-        task: str = "text-generation",
+        task: str | None = "text-generation",
         torch_dtype: torch.dtype = torch.float16,
+        **kwargs,
     ) -> None:
-        super().__init__(model_name_or_path, task, torch_dtype)
+        super().__init__(model_name_or_path, task, torch_dtype, **kwargs)
```

Comment:
**Suggestion:** Avoid using magic numbers. Instead, define them as constants at the top of your file or pass them as arguments to your function.
```suggestion
    DEFAULT_TASK = "text-generation"
    DEFAULT_TORCH_DTYPE = torch.float16

    def __init__(
        self,
        model_name_or_path: str,
        task: str | None = DEFAULT_TASK,
        torch_dtype: torch.dtype = DEFAULT_TORCH_DTYPE,
        **kwargs,
    ) -> None:
        super().__init__(model_name_or_path, task, torch_dtype, **kwargs)
```